LayoutTests/media/modern-media-controls/overflow-support/dropped-when-narrow.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true ] -->
+<meta name="viewport" content="width=device-width">
+<script src="../resources/media-controls-utils.js"></script>
+<script src="../../../resources/ui-helper.js"></script>
+<script src="../../../resources/js-test-pre.js"></script>
+<body>
+<audio src="../../content/test.mp4" style="position: absolute; left: 0; top: 0; width: 600px;" controls autoplay></video>
+<script type="text/javascript">
+
+window.jsTestIsAsync = true;
+
+description("Testing the <code>OverflowSupport</code> button does not appear when controls are narrow.");
+
+const media = document.querySelector("audio");
+const shadowRoot = window.internals.shadowRoot(media);
+
+media.addEventListener("play", function() {
+    media.pause();
+
+    shouldBecomeDifferent("shadowRoot.querySelector('button.overflow')", "null", () => {
+        shouldBecomeDifferent("shadowRoot.querySelector('button.overflow').getBoundingClientRect().width", "0", async () => {
+            media.style.width = "275px";
```

Comment:
this seems oddly specific and potentially fragile if the visual style (e.g. margins) of the media controls changes in the future.  is there any way we can do this that doesnt rely on a hardcoded `width`?

Reply:
Yeah, the number could be a different value too, but this is one that corresponds to the current state of how metrics work for media controls _right now_. Alas, I cannot think off the top of my head of a better technique, though I'd be happy to revisit after this initial change is committed. I will however make it a lower value, `200` works fine for instance (still fails prior to this change).